### ISSUE TITLE
The icons make the lists unreadable

### ISSUE THREAD

**Issue by GregOriol** [rodolpheh/awesome-matrix#1]
Maybe put them after the name? and in parenthesis? or in a table? or remove completely the icons and replace by the name?

**Comment by rodolpheh** [comment]
I agree with you but I'd like to keep it as a list so a table is out of sight. Ideally, I would make them aligned using tabs but Markdown is not really allowing this. Must find another idea. Maybe small images (16x16) with transparent images for alignment ?
